src/tests/golden/testdata/inflector/lib/Doctrine/Inflector/CachedWordInflector.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-declare(strict_types=1);
+declare(strict_types = 1);
```

Comment:
changes to test data formatting only pollute the PR, making it difficult to review

Reply:
Reply is the same as to the previous comment